packages/dapp/pages/index.tsx

```diff
@@ -17,6 +17,9 @@ const index: FC = (): JSX.Element => {
   useEffectAsync(async () => {
     if (managedContracts != null) {
       try {
+        //DEBUG
+        console.log(managedContracts?.dollarToken.address, "MANAGED DOLLAR");
+        console.log(managedContracts);
```

Comment:
& hook fetching ALL the addresses at return

https://github.com/ubiquity/ubiquity-dollar/blob/c28cae99bd525b45fbcc7da9539966ca79c8f2b5/packages/dapp/components/lib/hooks/contracts/use-manager-managed.tsx#L82